structure_tests/types/v1/structure.go

```diff
@@ -43,51 +52,37 @@ func (st StructureTest) RunAll(t *testing.T) int {
 	return testsRun
 }
 
-func (st StructureTest) RunCommandTests(t *testing.T) int {
+func (st *StructureTest) RunCommandTests(t *testing.T) int {
 	counter := 0
 	for _, tt := range st.CommandTests {
 		t.Run(tt.LogName(), func(t *testing.T) {
 			validateCommandTest(t, tt)
-			for _, setup := range tt.Setup {
-				utils.ProcessCommand(t, tt.EnvVars, setup, tt.ShellMode, false)
-			}
+			driver := st.NewDriver()
+			vars := append(tt.EnvVars, st.GlobalEnvVars...)
```

Comment:
Nice! I think we might want the order to go the other way though, so EnvVars will override GlobalEnvVars in the case of a duplicate.

Actually, I'm not sure how that would be handled. We should check and make sure the order is right here.

Reply:
so based on some testing, if I pass `--env "FOO=BAR" --env "FOO=BAZ"` to docker run and check the env, I'll see "FOO=BAZ"; so whichever the last value passed is is the one it will be set to in the container. so since we want the local envvars to override the global ones, I'll swap the order here.